features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/NewSessionScenario.kt

```diff
@@ -28,30 +30,32 @@ internal class NewSessionScenario(config: Configuration,
         val thread = HandlerThread("HandlerThread")
         thread.start()
 
-        Handler(thread.looper).post(Runnable {
-            // send 1st exception which should include session info
-            client.startSession()
-            Log.d("Bugsnag - New", "First session started")
-            Thread.sleep(SLEEP_MS)
-            flushAllSessions()
-            Log.d("Bugsnag - New", "First session flushed")
-            Thread.sleep(SLEEP_MS)
-            client.notify(generateException())
-            Log.d("Bugsnag - New", "First exception notified")
-            Thread.sleep(SLEEP_MS)
-            // stop tracking the existing session
-            client.pauseSession()
-            Log.d("Bugsnag - New", "First session paused")
-            Thread.sleep(SLEEP_MS)
-            // send 2nd exception which should contain new session info
-            client.startSession()
-            Log.d("Bugsnag - New", "Second session started")
-            Thread.sleep(SLEEP_MS)
-            flushAllSessions()
-            Log.d("Bugsnag - New", "Second session flushed")
-            Thread.sleep(SLEEP_MS)
-            client.notify(generateException())
-            Log.d("Bugsnag - New", "Second exception notified")
-        })
+        Handler(thread.looper).post(
+            Runnable {
+                // send 1st exception which should include session info
+                client.startSession()
+                Log.d("Bugsnag - New", "First session started")
+                Thread.sleep(SLEEP_MS)
+                flushAllSessions()
+                Log.d("Bugsnag - New", "First session flushed")
+                Thread.sleep(SLEEP_MS)
+                client.notify(generateException())
+                Log.d("Bugsnag - New", "First exception notified")
+                Thread.sleep(SLEEP_MS)
+// stop tracking the existing session
```

Comment:
Surprised that comments with no indentation are desirable/allowed?

Reply:
I don't believe there are indentation rules for detekt/ktlint by default, so have amended the indentation here manually.